### PR TITLE
feat: show container connection type and endpoint

### LINKED ISSUE #4727: Show connection endpoint instead of provider information for containers
### Is your enhancement related to a problem? Please describe

Currently the provider information is shown for containers, where the connection information is shown for Kubernetes.

This makes it somewhat confusing, when they are shown side by side or when connection name does not match provider.

Like here:

* https://github.com/containers/podman-desktop/pull/4539

lima-provider-both

* the version is missing: https://github.com/containers/podman-desktop/pull/3748
  (it is only showing the CLI version anyway, so it is [obsolete](https://github.com/containers/podman-desktop/pull/4503))

### Describe the solution you'd like

It would be better if the resources showed the endpoints in a similar matter (with the URL), for both types of connections.

And the provider information should probably move to the "left hand side", and not be repeated for every connection ?

podman-desktop-provider-endpoint

Note: this is just showing the raw values (type and socketPath), and has some additional features like name and version

The provider type should be Capitalized, and one might consider truncating long URLs (or only show the path.basepath)

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

----

PoC:

```html
              <div class="mt-2">
                <div class="text-gray-700 text-xs">{container.type} endpoint</div>
                <div class="mt-1">
                  <span class="my-auto text-xs" class:text-gray-900="{container.status !== 'started'}"
                    >unix://{container.endpoint.socketPath}</span>
                </div>
```
```html
              <div class="mt-2">
                <div class="text-gray-700 text-xs">Kubernetes endpoint</div>
                <div class="mt-1">
                  <span class="my-auto text-xs" class:text-gray-900="{kubeConnection.status !== 'started'}"
                    >{kubeConnection.endpoint.apiURL}</span>
                </div>
              </div>
```

### PR DESCRIPTION
**Pull request by afbjorklund**
Make the container and the kubernetes endpoint look more similar.

Show the connection type, since it might not match the provider.

### Screenshot/screencast of this PR

pd-endpoints

### What issues does this PR fix or reference?

Fixes #4727

### CODE REVIEW

File: packages/renderer/src/lib/preferences/PreferencesResourcesRendering.svelte [hunk 1]
```diff
@@ -473,6 +473,15 @@ function hasAnyConfiguration(provider: ProviderInfo) {
                   <ConnectionErrorInfoButton status="{status}" />
                 {/if}
               </div>
+              <div class="mt-2">
+                <div class="text-gray-700 text-xs">
+                  {#if container.type === 'docker'}Docker{:else if container.type === 'podman'}Podman{/if} endpoint
+                </div>
+                <div class="mt-1">
+                  <span class="my-auto text-xs" class:text-gray-900="{container.status !== 'started'}"
+                    >unix://{container.endpoint.socketPath}</span>
```
Comment (benoitf) [inline packages/renderer/src/lib/preferences/PreferencesResourcesRendering.svelte]: I think it won't work on windows as it's using `npipe:`

Comment (afbjorklund) [reply packages/renderer/src/lib/preferences/PreferencesResourcesRendering.svelte]: Seems to be copied everywhere, no function.

```
extensions/compose/src/templates/podman-compose.bat.mustache:set DOCKER_HOST=npipe://{{{ socketPath }}}
extensions/compose/src/templates/podman-compose.sh.mustache:export DOCKER_HOST=unix://{{{ socketPath }}}
```

Will probably make it harder to add `ssh:` etc.

Comment (afbjorklund) [reply packages/renderer/src/lib/preferences/PreferencesResourcesRendering.svelte]: Probably would be good with an abstraction for type as well

Comment (afbjorklund) [reply packages/renderer/src/lib/preferences/PreferencesResourcesRendering.svelte]: Changed back from URL to path, until the interface changes.

**Review by benoitf — APPROVED** [review]
probably need to have component tests to see if we're displaying correct container type and socketPath based on the status ?

**Review by jeffmaury — APPROVED** [review]
LGTM

**Comment by slemeur** [comment]
Merging for being part of 1.6.0 release.
Expecting follow up PR with tests.

**Pull request merged**